docs/_snippets/features/read-only.js

```diff
@@ -10,15 +10,57 @@ import { CS_CONFIG } from '@ckeditor/ckeditor5-cloud-services/tests/_utils/cloud
 ClassicEditor
 	.create( document.querySelector( '#snippet-read-only' ), {
 		cloudServices: CS_CONFIG,
+		toolbar: {
+			items: [
+				'exportPdf',
+				'exportWord',
+				'|',
+				'heading',
+				'|',
+				'bold',
+				'italic',
+				'link',
+				'numberedList',
+				'bulletedList',
+				'|',
+				'outdent',
+				'indent',
+				'|',
+				'uploadImage',
+				'blockQuote',
+				'insertTable',
+				'findAndReplace',
+				'undo',
+				'redo'
+			]
+		},
 		ui: {
 			viewportOffset: {
 				top: window.getViewportTopOffsetConfig()
 			}
+		},
+		exportPdf: {
+			stylesheets: [
+				'EDITOR_STYLES'
+			],
+			fileName: 'export-pdf-demo.pdf',
+			appID: 'cke5-docs',
+			converterOptions: {
+				format: 'A4',
+				margin_top: '20mm',
+				margin_bottom: '20mm',
+				margin_right: '12mm',
+				margin_left: '12mm',
+				page_orientation: 'portrait'
+			},
```

Comment:
This configuration doesn't reflect the way the demo is styled, so the exported document is not 1:1. Additionally, it misses content styles. While adding Export features to the editor, I think we should present it the way it should look like, so the content of the demo in the exported document should look like the one in the editor.

Reply:
Fixed.